### PR TITLE
Support for ZS sub-gamemodes and new configs

### PR DESCRIPTION
**Pull request by Sigilmare**
Fixed a bug where bots in ZS sub-gamemodes would pick banned zombie classes
Added 2 new configuration options in sv_config.lua because customization is great
Updated README

### CODE REVIEW

File: lua/d3bot/sv_zs_bot_handler/supervisor.lua [hunk 1]
```diff
@@ -154,7 +156,7 @@ function D3bot.DoNodeTrigger()
 		local nodeOrNil = D3bot.MapNavMesh:GetNearestNodeOrNil(ent:GetPos()) -- TODO: Don't call GetNearestNodeOrNil that often
 		if nodeOrNil then
 			if not D3bot.DisableNodeDamage and type(nodeOrNil.Params.DMGPerSecond) == "number" and nodeOrNil.Params.DMGPerSecond > 0 then
-				ent:TakeDamage(nodeOrNil.Params.DMGPerSecond*2, game.GetWorld(), game.GetWorld())
+				ent:TakeDamage(nodeOrNil.Params.DMGPerSecond, game.GetWorld(), game.GetWorld())
```
Comment (Dadido3) [inline lua/d3bot/sv_zs_bot_handler/supervisor.lua:159]: The damage taken every interval should be scaled to the interval duration.

(`ent:TakeDamage(nodeOrNil.Params.DMGPerSecond * D3bot.NodeDamageInterval, ...`)

**Comment by Dadido3** [comment]
Just change the damage scaling, otherwise this looks fine.

Also, welcome back HordeRally, long time not seen :D

File: lua/d3bot/sv_zs_bot_handler/supervisor.lua (hunk 1 above)
Comment (Sigilmare) [reply lua/d3bot/sv_zs_bot_handler/supervisor.lua:159]: Ah yea! Forgot about that, that makes a whole lot more sense.

**Comment by Sigilmare** [comment]
Yeah man! It's been very long, glad to be back haha

**Pull request merged**